apps/frontend/src/services/admin/user/AdminUserService.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  type CreateUserRequest,
+  GetAllUsersResponseSchema,
+  GetUserResponseSchema,
+  type PaginationQuery,
+  type UpdateUserRequest,
+} from "@repo/shared/schemas"
```

Comment:
nit: isn't this supposed to be
```suggestion
  type UpdateUserRequest,
} from "@repo/shared"
```